e2e/app/eoa/fund.go

```diff
@@ -81,10 +109,22 @@ func (t FundThresholds) MinBalance() *big.Int {
 }
 
 func (t FundThresholds) TargetBalance() *big.Int {
-	gwei := t.targetEther * params.GWei
+	gwei := t.targetETH * params.GWei
 	if gwei < 1 {
 		panic("ether float64 must be greater than 1 Gwei")
 	}
 
 	return math.NewInt(params.GWei).MulRaw(int64(gwei)).BigInt()
 }
+
+func convert(threshold FundThresholds, token tokens.Token) (FundThresholds, error) {
+	conversion, ok := tokenConversion[token]
+	if !ok {
+		return FundThresholds{}, errors.New("fund conversion", "token", token.String())
+	}
+
+	return FundThresholds{
+		minETH:    threshold.minETH * conversion,
```

Comment:
I don't think this is nicest thing to do because we are overwritting ETH with Omni, but the alternative is to implement Balance that can either be OMNI and ETH and then have conversions between those, which for this point where we only have ETH and OMNI was a bit overkill, I didn't want to generalise it to just `minBalance` because it's not clear then in which currency we define it above in the funding thresholds